PennMobile/Home/Model/HomeItemTypes.swift

```diff
@@ -23,6 +23,8 @@ final class HomeItemTypes: ModularTableViewItemTypes {
     let reservations: HomeCellItem.Type = HomeReservationsCellItem.self
     let courses: HomeCellItem.Type = HomeCoursesCellItem.self
     let gsrLocations: HomeCellItem.Type = HomeGSRLocationsCellItem.self
+    let invites: HomeCellItem.Type =  HomeGroupInvitesCellItem.self
+    //let pendingInvites: HomeCellItem.Type = HomePendingInvitesCellItem.self
```

Comment:
What is a `pendingInvites` cell? Is it different from the `invites` cell? If not, let's delete this comment.